raiden/tests/test_callbacks.py

```diff
@@ -20,20 +16,20 @@
 def test_direct_transfer_callback(raiden_network):
     app0, app1 = raiden_network  # pylint: disable=unbalanced-tuple-unpacking
 
-    messages = setup_messages_cb()
-    mlogger = MessageLogger()
-
-    a0_address = pex(app0.raiden.address)
-    a1_address = pex(app1.raiden.address)
+    # @ezdac: These vars are not used anywhere. Please address them
```

Comment:
@ezdac I have marked some regions in the test were there were unused variables. Please either use them in the test or remove them.